lab_1/start.py

```diff
@@ -1,18 +1,37 @@
-"""
-Concordance implementation starter
-"""
-
-from main import read_from_file
-import os
-
-if __name__ == '__main__':
-    #  use data.txt file to test your program
-    current_dir = os.path.dirname(os.path.abspath(__file__))
-    data = read_from_file(os.path.join(current_dir, 'data.txt'))
-    stop_words = []
-
-    #  here goes your logic: calling methods from concordance.py
-
-    RESULT = None
-    # DO NOT REMOVE NEXT LINE - KEEP IT INTENTIONALLY LAST
-    assert RESULT == [(), ()], 'Concordance not working'
+"""
+Concordance implementation starter
+"""
+import os
+from main import read_from_file, get_stop_words, calculate_frequencies
+from main import tokenize, remove_stop_words, get_top_n_words, write_to_file
+from main import get_concordance, get_adjacent_words, sort_concordance
+
+
+if __name__ == '__main__':
+    #  use data.txt file to test your program
+    current_dir = os.path.dirname(os.path.abspath(__file__))
+    data = read_from_file(os.path.join(current_dir, 'data.txt'))
+    stop_words = []
+
+    #  here goes your logic: calling methods from concordance.py
+    tokens = tokenize(data) # токенезирование
+    stop_words = get_stop_words()
+    write_to_file(os.path.join(current_dir, 'lab_1/stop_words.txt'), stop_words)
```

Comment:
No need for `lab_1`, the following will be enough:
`os.path.join(current_dir, 'stop_words.txt')`